test/StreamJsonRpc.Tests/AsyncEnumerableTests.cs

```diff
@@ -456,6 +456,7 @@ public async Task NotifyAsync_ThrowsIfAsyncEnumerableSent()
     public async Task ArgumentEnumerable_ReleasedOnErrorResponse()
```

Comment:
How does call scoped marshalable objects work with async enumerators? Since the call is technically completed when AsyncEnumerable is returned but I assume in most cases, the call scoped marshaled argument would be used while enumerator is active.

Reply:
Hmmmmm.... A very good question that's worth documenting, testing, and ... finding the answer to because I don't know. 😲

Reply:
It didn't work as you hoped. But I update it to support it.